### ISSUE TITLE
[bug]: NeoTree position can only be `left` or `right`

### ISSUE THREAD

**Issue by Kamyil** [shortcuts/no-neck-pain.nvim#400]
### Description

When setting Neotree integration configuration like this
```
    -- Supported integrations that might clash with `no-neck-pain.nvim`'s behavior.
    --- @type table
    integrations = {
      -- By default, if NeoTree is open, we will close it and reopen it when enabling the plugin,
      -- this prevents having the side buffers wrongly positioned.
      -- @link https://github.com/nvim-neo-tree/neo-tree.nvim
      NeoTree = {
        position = 'float',
        -- When `true`, if the tree was opened before enabling the plugin, we will reopen it.
        reopen = true,
      },
    },
```

Neovim will start with following error

```
NeoTree position can only be `left` or `right`
```

which is not really true, since neo-tree.nvim now supports a lot more position options
https://github.com/nvim-neo-tree/neo-tree.nvim?tab=readme-ov-file#position
(or maybe I'm missing something)

btw. super cool plugin, I'm enjoying it quite a lot today :D

### Neovim version

0.10.x

### Steps to reproduce

Set opts for integrations like this
```
    integrations = {
      -- By default, if NeoTree is open, we will close it and reopen it when enabling the plugin,
      -- this prevents having the side buffers wrongly positioned.
      -- @link https://github.com/nvim-neo-tree/neo-tree.nvim
      NeoTree = {
        position = 'float',
        -- When `true`, if the tree was opened before enabling the plugin, we will reopen it.
        reopen = true,
      },
    },
    ```
    and then reopen Neovim

### Relevant log output

```shell
.../nvim/lazy/no-neck-pain.nvim/lua/no-neck-pain/config.lua:400: NeoTree position can only be `left` or `right`

# stacktrace:
  - /no-neck-pain.nvim/lua/no-neck-pain/config.lua:400 _in_ **defaults**
  - /no-neck-pain.nvim/lua/no-neck-pain/config.lua:459 _in_ **setup**
  - /no-neck-pain.nvim/lua/no-neck-pain/init.lua:80 _in_ **setup**
  - init.lua:34
```

### Self-service

- [X] I'd be willing to fix this bug myself.

**Comment by shortcuts** [comment]
Hey @Kamyil, thanks for using the plugin and reporting the issue!

This is a leftover from the legacy implementation, I'll double check but right now we should only care about left or right, the other positions we can ignore them

**Comment by shortcuts** [comment]
Hey, I've released the fix in 2.0.4!

**Comment by Kamyil** [comment]
Wow, that was quick :o 
Thank you so much! you're rockin'
